tests/tools/oecert/host/host.cpp

```diff
@@ -108,261 +121,66 @@ static oe_result_t _generate_certificate(
-        FILE* output = NULL;
-        fopen_s(&output, evidence_filename, "wb");
-        if (!output)
-            OE_RAISE_MSG(
-                OE_FAILURE,
-                "Failed to open evidence file %s\n",
-                evidence_filename);
-        fwrite(evidence, evidence_size, 1, output);
-        fclose(output);
-    }
-
-    // Verify evidence
-    {
-        oe_claim_t* claims = NULL;
-        size_t claims_length = 0;
-
-        OE_CHECK(oe_verifier_initialize());
-
-        OE_CHECK_MSG(
-            oe_verify_evidence(
-                NULL,
-                evidence,
-                evidence_size,
-                endorsements,
-                endorsements_size,
-                NULL,
-                0,
-                &claims,
-                &claims_length),
-            "Failed to verify evidence. result=%u (%s)\n",
-            result,
-            oe_result_str(result));
-
-        OE_CHECK(oe_free_claims(claims, claims_length));
-        OE_CHECK(oe_verifier_shutdown());
-    }
-
-    // Write endorsements
-    if (endorsements_filename)
-    {
-        FILE* endorsements_file;
-        fopen_s(&endorsements_file, endorsements_filename, "wb");
-        if (!endorsements_file)
-            OE_RAISE_MSG(
-                OE_FAILURE,
-                "Failed to open endorsements file %s\n",
-                endorsements_filename);
-
-        fwrite(endorsements, endorsements_size, 1, endorsements_file);
-        fclose(endorsements_file);
-        printf("endorsements_size = %zu\n", endorsements_size);
-    }
-
-    result = OE_OK;
-
-done:
-    return result;
-}
-
 static void _display_help(const char* cmd)
 {
     printf("Usage: %s ENCLAVE_PATH Options\n", cmd);
     printf("\tOptions:\n");
     printf(
-        "\t%s PRIVKEY PUBKEY: generate der remote attestation "
+        "\t%s <privkey> <pubkey> : generate der remote attestation "
         "certificate.\n",
         INPUT_PARAM_OPTION_CERT);
     printf(
-        "\t%s : generate binary enclave report and collateral.\n",
-        INPUT_PARAM_OPTION_REPORT);
+        "\t%s : generate binary enclave report\n", INPUT_PARAM_OPTION_REPORT);
     printf(
         "\t%s : generate binary enclave evidence.\n",
         INPUT_PARAM_OPTION_EVIDENCE);
-    printf("\t%s : output filename.\n", INPUT_PARAM_OPTION_OUT_FILE);
     printf(
-        "\t%s : file for endorsements (use with %s).\n",
+        "\t%s <output filename> : file for certificate, report or evidence "
+        "(default: %s)\n",
+        INPUT_PARAM_OPTION_OUT_FILE,
+        DEFAULT_OUT_FILE);
+    printf(
+        "\t%s : file for endorsements (use with %s or %s).\n",
         INPUT_PARAM_OPTION_ENDORSEMENTS_FILENAME,
+        INPUT_PARAM_OPTION_REPORT,
         INPUT_PARAM_OPTION_EVIDENCE);
+    printf(
+        "\t%s <log filename> (default: %s)\n",
+        INPUT_PARAM_OPTION_LOG_FILE,
+        DEFAULT_LOG_FILE);
+    printf("\t%s\n", INPUT_PARAM_VERBOSE);
 
     // TODO: Add option to display certs
```

Comment:
These 2 TODOs can be removed. We have option to display certs. We no longer pursue PEM version of the certs.

Reply:
Got it. Removed.